File: URI/1040.py
lista = input().split()

media = (float(lista[0])*2 + float(lista[1])*3 + float(lista[2])*4 + float(lista[3]))/10
print("Media: {:.1f}".format(media))

if media >= 7.0:
	print("Aluno aprovado.")
elif media < 5.0:
	print("Aluno reprovado.")
else:
	print("Aluno em exame.")
	exame = float(input())
	print("Nota do exame: {:.1f}".format(exame))

	media = (media + exame)/2
	if media >=5.0:
		print("Aluno aprovado.")
	else:
		print("Aluno reprovado.")

	print("Media final: {:.1f}".format(media))

File: URI/C/1040.c
#include<stdio.h>
#include<stdlib.h>

	float N1, N2, N3, N4, media, exam;

void main(){
	scanf("%f %f %f %f",&N1,&N2,&N3,&N4);
	media=(N1*2+N2*3+N3*4+N4*1)/10;
	printf("Media: %.1f\n", media);
	if(media>7.0)
		printf("Aluno aprovado.\n");
	if(media<5.0)
		printf("Aluno reprovado.\n");
	if(media>=5.0 && media<=6.9){
		printf("Aluno em exame.\n");
		scanf("%f",&exam);
		printf("Nota do exame: %.1f\n", exam);
		media=(media+exam)/2;
		
		if(media>=5.0)
			printf("Aluno aprovado.\nMedia final: %.1f\n",media);
		else
			printf("Aluno reprovado.\nMedia final: %.1f\n",media);
	}
	
	
}
